tests/test_feature.py

```diff
@@ -286,6 +287,21 @@ def test_process_folder(tmpdir):
     assert all(y.index.levels[2] == index.levels[1])
     np.testing.assert_array_equal(y.values, y_expected)
 
+    # non-existing folder
+    with pytest.raises(FileNotFoundError):
+        feature.process_folder('bad-folder')
+
+    # empty folder
+    root = str(tmpdir.mkdir('empty'))
+    y = feature.process_folder(root)
+    pd.testing.assert_frame_equal(
+        y,
+        pd.DataFrame(
+            dtype=object,
```

Comment:
Strange we need `object` here, I was expecting `float`...

Reply:
I get:

```python
>>> pd.Series()
<ipython-input-101-fa15c544da3b>:1: DeprecationWarning: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning.
  pd.Series()
Out[101]: Series([], dtype: float64)
>>> pd.Series(dtype=object)
Series([], dtype: object)
>>> pd.Series(dtype=float)
Series([], dtype: float64)
```
So, I guess there is something wrong (or is it a feature?) when converting the series to a dataframe.
If I do
```python
print(y)
print(y.dtypes)
```
I get
```
Empty DataFrame
Columns: [o1-0, o2-0, o3-0, o1-1, o2-1, o3-1]
Index: []
o1-0    object
o2-0    object
o3-0    object
o1-1    object
o2-1    object
o3-1    object
dtype: object
```

Reply:
>  So it seems they will only change the default type to float for Series in the future.

No, they will change it to `object`:

> The default dtype for empty Series will be 'object' 

Do you think we should do the same then?

Reply:
Ah, I see. So they actually change the default behavior of `Series` to that of `DataFrame`. That makes sense of course!

> Do you think we should do the same then?

Mhh, good question. Probably it makes indeed sense to go with `object`, since this is most generic data type.

Reply:
I created https://github.com/audeering/audinterface/issues/35

Reply:
Why not directly change it here, we have to break one behavior anyway, at the moment you decided to change the dataframe to `float`. What about other places where we return an empty dataframe?

Reply:
> Why not directly change it here

So you mean changing to `object` when we return an empty `Series` and do not return `DataFrame` as `float`, correct?

> What about other places where we return an empty dataframe?

As far as I saw there's only one place, but I will check again.

Reply:
> So you mean changing to object when we return an empty Series and do not return DataFrame as float, correct?

Yes, exactly.

Reply:
Done, this basically reverts some of the changes from #32